src/libpanic_unwind/gcc.rs

```diff
@@ -87,10 +87,8 @@ pub fn payload() -> *mut u8 {
 }
 
 pub unsafe fn cleanup(ptr: *mut u8) -> Box<dyn Any + Send> {
-    let my_ep = ptr as *mut Exception;
-    let cause = (*my_ep).cause.take();
-    uw::_Unwind_DeleteException(ptr as *mut _);
```

Comment:
Who calls DeleteException on the exception? There's not a drop impl on `Exception` or `_Unwind_Exception` AFAICT.

Reply:
All `_Unwind_DeleteException` does it invoke the call the `exception_cleanup` callback in `_Unwind_Exception`. Since the exception is our own, we can just free the `Box` directly.

Reply:
I'm flagging this as temporarily unresolved to leave a comment here too since this was a question I had. Could a comment be left indicating that we're explicitly not calling `_Unwind_DeleteException`? Also, do you have a link to the source that it only calls the internal callback?

Reply:
The [documentation](https://refspecs.linuxbase.org/LSB_1.3.0/gLSB/gLSB/baselib--unwind-deleteexception.html) for `_Unwind_DeleteException` explicitly says that this function is for foreign exception that the runtime does not know how to free.

Reply:
Oh sure sorry I'm not questioning the validity of this deletion, just un-resolving it so my request for a comment in the code didn't show up as collapsed.